en/setup/_gcc_toolchain_installation.txt

```diff
@@ -40,15 +26,8 @@ arm-none-eabi-gcc --version
 The output should be something similar to:
 
 ```sh
-arm-none-eabi-gcc (GNU Tools for ARM Embedded Processors) 5.4.1 20160609 (release) [ARM/embedded-5-branch revision 237715]
-Copyright (C) 2015 Free Software Foundation, Inc.
+arm-none-eabi-gcc (GNU Tools for Arm Embedded Processors 7-2017-q4-major) 7.2.1 20170904 (release) [ARM/embedded-7-branch revision 255204]
+Copyright (C) 2017 Free Software Foundation, Inc.
 This is free software; see the source for copying conditions.  There is NO
 warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 ```
-
```

Comment:
Removed this section as it makes no sense now the 32bit support libraries are not needed. Actually the whole section technically isn't needed since it no longer provides any real troubleshooting - however telling people how to check versions is always useful step in debuggin.